srcpkgs/hugo/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'hugo'
 pkgname=hugo
-version=0.101.0
+version=0.102.0
```

Comment:
ah sorry i thought this was bumping 102 to 102.1, it's fine to go to .1 i guess

Reply:
No worries. :) I'll change it back to 0.102.1.